src/sass/cloudfrontinvalidation.scss

```diff
@@ -0,0 +1,11 @@
+.cloudfrontinvalidation_message {
+    word-break: break-word;
+    max-width: 800px;
```

Comment:
instead of 800px; use something like units('desktop');

Reply:
Updated